lms/static/js/instructor_dashboard/student_admin.js

```diff
@@ -483,6 +490,60 @@
             });
         };
 
+        StudentAdmin.prototype.override_problem_score_single = function() {
+            var defaultErrorMessage, fullDefaultErrorMessage, fullSuccessMessage,
+                problemToReset, score, sendData, successMessage, uniqStudentIdentifier,
+                that = this;
+            uniqStudentIdentifier = this.$field_student_select_grade.val();
+            problemToReset = this.$field_problem_select_single.val();
+            score = this.$field_select_score_single.val();
+            if (!uniqStudentIdentifier) {
+                return this.$request_err_grade.text(
+                    gettext('Please enter a student email address or username.')
+                );
+            }
+            if (!problemToReset) {
+                return this.$request_err_grade.text(
+                    gettext('Please enter a problem location.')
```

Comment:
Same as the last- this is the error message the user sees when they have not entered a problem location for a student admin task. I copied it from the other tasks.